setup.cfg

```diff
@@ -0,0 +1,2 @@
+[flake8]
+max-line-length=150
```

Comment:
better keep this something like 80 or 120 and remove the test data from your test code files